config/settings.py

```diff
@@ -61,6 +108,12 @@
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
 ]
 
+# START_FEATURE debug_toolbar
+if DEBUG_TOOLBAR:
+    INSTALLED_APPS.append('debug_toolbar')
+    MIDDLEWARE.append('debug_toolbar.middleware.DebugToolbarMiddleware')
+# END_FEATURE debug_toolbar
```

Comment:
Hmmm, suggestion: split this up into two conditionals so that the appends live near what they're appending to. It does make this code more verbose, but I think it's generally better to colocate the lists being changed.